client/src/templates/Challenges/redux/create-question-epic.js

```diff
@@ -61,31 +58,21 @@ function createQuestionEpic(action$, state$, { window }) {
         ${projectFormValues
           ?.map(([key, val]) => `${key}: ${val}\n`)
           ?.join('') || filesToMarkdown(files)}
-        \n
+
         ${endingText}`
       );
 
       const altTextMessage = dedent(
         `**Tell us what's happening:**
-
-
-
```

Comment:
Did you mean to remove these empty lines?

Reply:
No, I just rushed copying from GitHub which does not copy blank lines.